src/ResourceManager/Sql/Commands.Sql/Az.Sql.psd1

```diff
@@ -239,7 +239,13 @@ CmdletsToExport = 'Get-AzSqlDatabaseTransparentDataEncryption',
     'Stop-AzSqlElasticJob',
     'Get-AzSqlElasticJobExecution',
     'Get-AzSqlElasticJobStepExecution',
-    'Get-AzSqlElasticJobTargetExecution'
+    'Get-AzSqlElasticJobTargetExecution',	               
+    'Get-AzureRmSqlElasticJobTargetExecution',
+    'Add-AzureRmSqlManagedInstanceKeyVaultKey',
+    'Get-AzureRmSqlManagedInstanceKeyVaultKey',
+    'Remove-AzureRmSqlManagedInstanceKeyVaultKey',
+    'Get-AzureRmSqlManagedInstanceTransparentDataEncryptionProtector',
+    'Set-AzureRmSqlManagedInstanceTransparentDataEncryptionProtector'
```

Comment:
All of these should have the "Az" prefix.  Please follow instructions here: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/create-alias-mappings.md

Reply:
Step 3 fails with multiple errors(~100) like following
Exception calling "Add" with "2" argument(s): "Item has already been added. Key in dictionary: 'Get-AzProviderOperation'  Key being added: 'Get-AzProviderOperation'"
At D:\WS\Azure\powershell\tools\CreateAliasMapping.ps1:23 char:13
+             $mapping[$name].Add($cmdletalias, $_)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentException

Reply:
Please try doing a git clean -xdf before running this command (be sure to commit your local changes before cleaning).  This script has been working for other partners and I can't repro that issue.

Reply:
Be sure to rename the cmdlets in the file before running that script again.